src/inspect.rs

```diff
@@ -197,7 +193,8 @@ impl MetadataPrinter {
         table.printstd();
 
         let text = format!("```yaml\n{rules_yaml}```");
-        self.render_markdown(&text, no_color)
+        self.render_markdown(&text, no_color);
+        Ok(())
```

Comment:
Same here, this method is now infallible

Reply:
We serialize the rules with serde though.